### PR TITLE
readme widget addition

### PR DESCRIPTION
**Pull request by ruvenzx**
## Related Issues

readme widget addition

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -391,6 +391,163 @@ export class AppRoutingModule {}
...
+- Custom fields also appear in the table.
+
+###### Usage
+
+`manage-users.component.html`
+
+```html
+<user-management
+	[tenant]="tenant"
+	[theme]="theme"
```
Comment (dorsha) [inline README.md:426]: No need for theme and debug (like we don't expose in other sdks)

**Review by dorsha — COMMENTED** [review]
Cool, @asafshen please review as well.

File: README.md [hunk 2]
```diff
@@ -391,6 +391,145 @@ export class AppRoutingModule {}
...
+@Component({
+	selector: 'app-manage-users',
+	templateUrl: './manage-users.component.html'
+})
+export class ManageUsersComponent {
+	tenant = 'tenantId';
+
+	constructor(private _: Router) {}
+}
+```
```
Comment (asafshen) [inline README.md:442]: you dont need this part, the html is enough

File: README.md [hunk 3]
```diff
@@ -391,6 +391,145 @@ export class AppRoutingModule {}
...
+import { Router } from '@angular/router';
+
+@Component({
+	selector: 'app-manage-roles',
+	templateUrl: './manage-roles.component.html'
+})
+export class ManageRolesComponent {
+	tenant = 'tenantId';
+
+	constructor(private _: Router) {}
```
Comment (asafshen) [inline README.md:484]: same as above

File: README.md [hunk 4]
```diff
@@ -391,6 +391,145 @@ export class AppRoutingModule {}
...
+```html
+<access-key-management
+	[tenant]="tenant"
+	widgetId="access-key-management-widget"
+/>
+```
+
+`manage-access-keys.component.ts`
+
+```js
```
Comment (asafshen) [inline README.md:514]: same as above

File: README.md [hunk 5]
```diff
@@ -391,6 +391,86 @@ export class AppRoutingModule {}
...
+- Delete an existing user
+
+Note:
+
+- Custom fields also appear in the table.
+
+###### Usage
+
+```html
+<user-management [tenant]="tenant" widgetId="user-management-widget" />
```
Comment (asafshen) [inline README.md:422]:
```suggestion
<user-management tenant="tenant-id" widgetId="user-management-widget" />
```

you don't need to bind it (aka `[...]`)

File: README.md [hunk 6]
```diff
@@ -391,6 +391,86 @@ export class AppRoutingModule {}
...
+
+Note:
+
+- The `Editable` field is determined by the user's access to the role - meaning that project-level roles are not editable by tenant level users.
+- You need to pre-define the permissions that the user can use, which are not editable in the widget.
+
+###### Usage
+
+```html
+<role-management [tenant]="tenant" widgetId="role-management-widget" />
```
Comment (asafshen) [inline README.md:446]:
```suggestion
<role-management tenant="tenant-id" widgetId="role-management-widget" />
```

File: README.md [hunk 7]
```diff
@@ -391,6 +391,86 @@ export class AppRoutingModule {}
...
+
+- Create a new access key
+- Activate / deactivate an existing access key
+- Delete an exising access key
+
+###### Usage
+
+```html
+<access-key-management
+	[tenant]="tenant"
```
Comment (asafshen) [inline README.md:466]:
```suggestion
	tenant="tenant-id"
```

**Review by asafshen — COMMENTED** [review]
some small comment about the tenant id

**Review by asafshen — APPROVED** [review]

**Pull request merged**